lms/templates/lti_launches/new_lti_launch.html.jinja2

```diff
@@ -1 +1,5 @@
-<iframe width="100%" height="100%" src="{{hypothesis_url}}" />
\ No newline at end of file
+{% for url in asset_urls("postmessage_json_rpc_server_js") %}
+  <script src="{{ url }}"></script>
+{% endfor %}
```

Comment:
When merging this locally I tripped myself up by putting the config _after_ the script and it failed to load.

Reply:
Is there a workaround? Starting the server should be triggered on document load or something?

Reply:
Not essential, but you could do that. This was just a heads up to avoid making the same mistake as me.